packages/node-runtime-worker-thread/src/child-process-evaluation-listener.ts

```diff
@@ -29,10 +31,10 @@ export class ChildProcessEvaluationListener {
           return workerRuntime.evaluationListener?.validateConfig?.(key, value) ?? Promise.resolve(null);
         },
         getConfig(key) {
-          return workerRuntime.evaluationListener?.getConfig?.(key) as any;
```

Comment:
OK apparently I didn't undo ALL the `as any` removals, but this seems to not matter, so I'll just leave it be for now.